src/components/common/MoveButton.tsx

```diff
@@ -127,16 +116,15 @@ const MoveButton = ({
         menuItemClassName={ITEM_MENU_MOVE_BUTTON_CLASS}
         iconClassName={ITEM_MOVE_BUTTON_CLASS}
       />
-
-      {items?.data && open && (
+      {itemIds && open && (
         <ItemSelectionModal
-          title={title}
+          titleKey={BUILDER.MOVE_ITEM_MODAL_TITLE}
           isDisabled={isDisabled}
           buttonText={buttonText}
           onClose={onClose}
```

Comment:
It looks like you do not reset the `itemIds` state when closing the dialog ?

Reply:
yes, I fixed it ! Thanks